docs/src/app/components/master.jsx

```diff
@@ -134,127 +145,223 @@ const Master = React.createClass({
     );
   },
 
-  _getTabs() {
+  _getSectionsData() {
+    return [
+      {
+        route: '/get-started',
+        title: 'Get Started',
+      },
+      {
+        route: '/customization',
+        title: 'Customization',
+      },
+      {
+        route: '/components',
+        title: 'Components',
+      },
+    ];
+  },
+
+  _getAppBar() {
+    const styles = this.getStyles();
+
+    let
+      title = null,
+      tabs = null,
+      waterfall,
+      position = 'fixed'
+      ;
+
+    if (this.state.renderTabs || !this.state.currentSection) {
+      title = (
+        <EnhancedButton
+          linkButton={true}
+          href="/#/home"
+          ref={el => { this.logoElement = ReactDom.findDOMNode(el); }}>
```

Comment:
Can't we use https://github.com/chenglou/react-motion to get ride of the ReactDOM?